sdk/storage/storage-file-datalake/tests.yml

```
@@ -5,3 +5,7 @@ extends:
   parameters:
     PackageName: "@azure/storage-file-datalake"
     TestSamples: false
+  EnvVars:
+    AZURE_CLIENT_ID: $(aad-azure-sdk-test-client-id)
+    AZURE_TENANT_ID: $(aad-azure-sdk-test-tenant-id)
+    AZURE_CLIENT_SECRET: $(aad-azure-sdk-test-client-secret)
```

The indentation for `EnvVars` is off which is why the yml file couldn't recognize that and the job failed as a result.
```suggestion
    ResourceServiceDirectory: storage
    TimeoutInMinutes: 90
    ResourceGroupLocation: canadacentral
    ArmTemplateParameters: "@{ enableVersioning = $true }"
    EnvVars:
      AZURE_CLIENT_ID: $(aad-azure-sdk-test-client-id)
      AZURE_TENANT_ID: $(aad-azure-sdk-test-tenant-id)
      AZURE_CLIENT_SECRET: $(aad-azure-sdk-test-client-secret)
```

Also adding the rest of the params from storage-blob configuration.